app/routes/docs.terms-of-service.md

```diff
@@ -0,0 +1,38 @@
+---
+handle:
+  breadcrumb: Terms of Service
+---
+
+# Terms of Service
+
+GCN Notices and Circulars are rapid communications which are by definition preliminary, and submitted as best-effort for the accuracy of their content. They are citable in publications, but are not peer reviewed.
+
+The sharing of preliminary analysis and results in an open and respectful manner enables the productive exchange of information and ideas. Notices and Circulars contain factual information on observations and analysis of astronomical transients, not opinions, speculation, or disagreements with individuals or groups. The Circulars [style guide](/circulars/styleguide) provides additional criteria and community standards for Circular content.
+
+## Retractions
+
+If an initial automated or manually-generated Notice or Circular is later deemed to be misclassified as an astronomical object, the generating team should promptly submit another Notice or Circular informing the community of the retracted claim.
```

Comment:
I am wondering whether we should go with grammar or naming convention throughout as Notices/Circulars.